docs-src/package.json

```diff
@@ -6,16 +6,16 @@
   "dependencies": {
     "@babel/plugin-proposal-object-rest-spread": "^7.5.5",
     "@babel/preset-env": "^7.5.5",
-    "@mdx-js/mdx": "^1.5.5",
```

Comment:
I think I want to try to unpin some of these versions and see if I can get it to build again and then remove them if I can